words.html

```diff
@@ -403,447 +407,445 @@
+gdy skończysz, kliknij:
 </p>
 <p id="distrust_4_note_2">
-(simulating: 10 rounds per match)
+(symulacja: 10 rund na mecz)
 </p>
 
 <p id="distrust_5">
-The same thing happens:
-with a lower "win-win" reward, <span class="all_d">Always Cheat</span> takes over.
-Game theory has two powerful ideas about this:
+Dzieje się to samo:
+z niższą nagrodą za wzajemną współpracę, <span class="all_d">Notoryczny Oszust</span> wygrywa.
+W teorii gier są dwie teorie na ten temat:
 <br><br>
-<b>"Zero-sum game".</b> This is the sadly common belief that a gain for "us"
-<i>must</i> come at a loss to "them", and vice versa.
+<b>"Gra o sumie zerowej".</b> To jest niestety powszechne przekonanie, że zysk dla "nas"
+<i>musi</i> oznaczać stratę dla "nich", i vice versa.
 <br><br>
-<b>"Non-zero-sum game".</b> This is when people make the hard effort
-to create a win-win solution! (or at least, avoid a lose-lose)
-Without the non-zero-sum game, <i>trust cannot evolve.</i>
+<b>"Gra o sumie niezerowej".</b> Ma miejsce gdy ludzie starają się
+uzyskać rozwiązanie korzystne dla wszystkich! (lub przynajmniej uniknąć rozwiązania dla wszystkich niekorzystnego)
+Bez gry o sumie niezerowej, <i>zaufanie nie może ewoluować.</i>
 <br><br>
-Speaking of which,
-let's now look at our third &amp; final barrier to the evolution of trust...
+Skoro o tym mowa,
+spójrzmy na trzecią i ostatnią barierę dla ewolucji zaufania...
 </p>
 <p id="distrust_5_btn">
-<s>Misteaks</s> Mistakes. &rarr;
+<s>Błendy</s> Błędy. &rarr;
 </p>
 
 <!-- - - - - - - - - - - - - - - - - -->
 <!-- - - - - - NOISE! - - - - - - - -->
 <!-- - - - - - - - - - - - - - - - - -->
 
 <p id="noise_1">
-As cool as <span class="tft">Copycat</span> is, it has a huge, fatal weakness I haven't mentioned yet.
-To understand the problem, let's say two <span class="tft">Copycats</span> are playing against each other:
+<span class="tft">Pauga</span> jest naprawdę super, ale ma wielką, fatalną słabość, o której jeszcze nie wspomniałem.
+Aby zrozumieć problem, załóżmy że dwie <span class="tft">Papugi</span> grają przeciwko sobie:
 </p>
 <p id="noise_1_end">
-Being "nice" players, both their first moves will be:
+Jako że są "miłymi" graczami, ich pierwsze ruchy będą:
 </p>
 
 <p id="noise_2">
-And normally, they'd just pay back each others' kindness and sing Kumbaya until the end of time.
+Jak dotąd po prostu obdarowywaliby się nawzajem życzliwością aż po kres czasów.
 </p>
 <p id="noise_2_end">
-But what if, while trying to reciprocate goodness...
+Ale co jeśli próbując odwzajemnić sie dobrocią...
 </p>
 
 <p id="noise_3">
-<b>OH NO</b>
+<b>O NIEE</b>
 <br>
-Mistakes, miscommunication, misinterpretations -- accidents happen all the time in real life.
+Pomyłki, nieporozumienia, błędne interpretacje — w prawdziwym życiu wypadki zdarzają się cały czas.
 </p>
 <p id="noise_3_end">
-But if the other person doesn't <i>think</i> it was an accident...
+Ale jeśli druga osoba <i>nie uzna</i>, że był to wypadek...
```

Comment:
Czy tylko ja jak to czytam, myślę o tym, że zaraz nas wsadzą w newsy o tym, że śmiejemy się ze smoleńskich teorii spiskowych?